index.html

```diff
@@ -68,11 +70,49 @@ <h2>Simplicity</h2>
             </div>
         </div>
 
+
+        <div class="about-content">
+            <h2>Meet the team</h2>
+            <div class="underline"></div>
+            <div class="all-member">
+                <div class="member">
+                    <img class="team-member" src="./images/pascal.jpg" alt="Pascal" />
+                    <p>Pascal Ulor</p>
+                    <h3>Team Lead</h3>
+                </div>
+                <div class="member">
+                    <img class="team-member" src="./images/mariam.jpg" alt="Mariam" />
+                    <p>Mariam Adedeji</p>
+                    <h3>Team Member</h3>
+                </div>
+                <div class="member">
+                    <img class="team-member" src="./images/justina.jpg" alt="Justina" />
+                    <p>Justina Vebra</p>
```

Comment:
"Justina" is a female name 😅

Reply:
By the way, I like the responsiveness of the team part. More work is needed on other parts though:

![Screenshot 2020-01-08 at 17 09 13](https://user-images.githubusercontent.com/230534/71989165-d073dc00-3239-11ea-8a08-ef45e5994304.png)